src/jiragit.ts

```diff
@@ -20,13 +20,16 @@ const DEFAULT_CONFIG: JiraConfig = {
     "TODO: generate in https://id.atlassian.com/manage-profile/security/api-tokens",
   host: "https://mycompany.atlassian.net",
   projectKey: "Example: for issue like ABC-123 ABC is the project key",
+  predicate:
```

Comment:
I think it's better to call it `jql` instead for easy understanding
```suggestion
  jql:
```

Reply:
Done.